app/src/main/java/com/android/feedme/ui/camera/Camera.kt

```diff
@@ -321,28 +280,3 @@ fun CameraPreview(controller: LifecycleCameraController, modifier: Modifier = Mo
       },
       modifier = modifier.testTag("CameraPreview"))
 }
-
-@Composable
-fun PhotoBottomSheetContent(bitmaps: List<Bitmap>, modifier: Modifier = Modifier) {
-  // Show a message if there are no photos
-  if (bitmaps.isEmpty()) {
-    Box(modifier = modifier.padding(16.dp), contentAlignment = Alignment.Center) {
-      Text(text = "There are no photos yet")
-    }
-  } else {
-    // Display the photos in a grid
-    LazyVerticalStaggeredGrid(
-        columns = StaggeredGridCells.Fixed(2),
-        horizontalArrangement = Arrangement.spacedBy(16.dp),
-        verticalItemSpacing = 16.dp,
-        contentPadding = PaddingValues(16.dp),
-        modifier = modifier) {
-          items(bitmaps) { bitmap ->
-            Image(
-                bitmap = bitmap.asImageBitmap(),
-                contentDescription = "Photo",
-                modifier = Modifier.clip(RoundedCornerShape(10.dp)))
-          }
-        }
-  }
-}
```

Comment:
So the local gallery feature was totally removed right ?

Reply:
Yes indeed, it was not needed now that we process pictures one by one